docs/fsharp/index.md

```diff
@@ -14,45 +13,69 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
+    0
+```
 
-Additionally, the [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference) is a great resource for learning about FSharp.Core, the core library which is a part of F#.
+F# is about productivity at its heart. The tooling support for F# is ubiquitous and full of advanced features.
 
-## The F# Software Foundation
+## Learning F# #
 
-Although Microsoft is the primary developer of the F# language and its tooling, F# is also backed by an independent foundation, the F# Software Foundation (FSSF).
+[Tour of F#](tour.md) gives an overview of major language features with lots of code samples. This is recommended if you are new to F# and want to get a feel for how the language works.
 
-The mission of the F# Software Foundation is to promote, protect, and advance the F# programming language, and to support and facilitate the growth of a diverse and international community of F# programmers.
+[Get started with F# in Visual Studio](get-started/get-started-visual-studio.md) if you're on Windows and want the full Visual Studio IDE experience.
+
+[Get started with F# in Visual Studio for Mac](get-started/get-started-with-visual-studio-for-mac.md) if you're on macOS and want to use a Visual Studio IDE.
+
+[Get Started with F# in Visual Studio Code](get-started/get-started-vscode.md) if you want a lightweight, cross-platform, and feature-packed IDE experience.
+
+[Get started with F# with the .NET Core CLI](get-started/get-started-command-line.md) if you just want a great set of command-line tools.
+
+## References
+
+[F# Language Reference](language-reference/index.md) is the official, comprehensive reference for all F# language features. Each article explains the syntax and shows code samples. You can use the filter bar in the table of contents to find specific articles.
+
+[F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference) is the API reference for the F# Core Library.
 
-To learn more and get involved, check out [fsharp.org](http://fsharp.org).
+## Additional guides
 
-## Documentation
+[F# for Fun and Profit](https://swlaschin.gitbooks.io/fsharpforfunandprofit/content/) is a comprehensive and very detailed book on learning F#. Its contents and author are beloved by the F# community.
 
-* [Tutorials](tutorials/getting-started/index.md)
-* [Functions as First-Class Values](introduction-to-functional-programming/functions-as-first-class-values.md)<!--[Introduction to Functional Programming](introduction-to-functional-programming/index.md)-->
-* [Language Reference](language-reference/index.md)
-* [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference)
+[F# Programming Wikibook](https://en.wikibooks.org/wiki/F_Sharp_Programming) is a wikibook about learning F#. It is also a product of the F# community.
```

Comment:
Who would be the target audience for the wikibook? That would be important for this audience.

Reading on, this comment applies to all the resources you mention: If you know the target audience, that would be useful to know.